packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -0,0 +1,127 @@
+.rt-RadioCardGroupRoot {
+  user-select: none;
+  cursor: default;
+}
+
+.rt-RadioCardGroupItem {
+  display: flex;
+  align-items: center;
+}
+
+/* Avoid unintentional drag interactions (e.g. on images) */
+.rt-RadioCardGroupItem * {
+  pointer-events: none;
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * SIZES                                                                                           *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-RadioCardGroupItem {
+  padding: var(--radio-card-group-item-padding);
+  border-radius: var(--radio-card-group-item-radius);
+  gap: var(--radio-card-group-item-gap);
+}
+
+@breakpoints {
+  .rt-RadioCardGroupRoot {
+    &:where(.rt-r-size-1) {
+      --radio-card-group-item-padding: var(--space-3);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-2);
+    }
+    &:where(.rt-r-size-2) {
+      --radio-card-group-item-padding: var(--space-4);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-3);
+    }
+    &:where(.rt-r-size-3) {
+      --radio-card-group-item-padding: var(--space-5);
+      --radio-card-group-item-radius: var(--radius-5);
+      --radio-card-group-item-gap: var(--space-4);
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * VARIANTS                                                                                        *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-RadioCardGroupItem {
+  position: relative;
+  /* pseudo element for focus outline */
+  &::before {
+    content: '';
+    display: block;
+    position: absolute;
+    inset: 0;
+    border-radius: inherit;
+  }
```

Comment:
Super nit, but it drives me insane that when trying to target the radio cards with inspector, the pseudo catches pointer events

Reply:
Yeah hate that too, nice catch!